### PR TITLE
MLPAB-2108 Log authentication events

### PR DESCRIPTION
**Pull request by hawx**

### CODE REVIEW

File: internal/telemetry/telemetry.go [hunk 1]
```diff
@@ -16,14 +17,12 @@ import (
...
 	semconv "go.opentelemetry.io/otel/semconv/v1.12.0"
 	"go.opentelemetry.io/otel/trace"
-	"google.golang.org/grpc"
 )
 
 func Setup(ctx context.Context, resource *resource.Resource) (func(context.Context) error, error) {
 	traceExporter, err := otlptracegrpc.New(ctx,
 		otlptracegrpc.WithInsecure(),
-		otlptracegrpc.WithEndpoint("0.0.0.0:4317"),
-		otlptracegrpc.WithDialOption(grpc.WithBlock()))
```
Comment (hawx) [inline internal/telemetry/telemetry.go:26]: this option is deprecated

File: internal/page/sign_out.go [hunk 2]
```diff
@@ -24,6 +24,8 @@ func SignOut(logger Logger, sessionStore SessionStore, oneLoginClient OneLoginCl
 			endSessionURL = redirectURL
 		}
 
+		logger.InfoContext(r.Context(), "logout")
```
Comment (acsauk) [inline internal/page/sign_out.go:27]: Could it be useful to attach a session ID here to match the login event? Thinking specifically if we wanted/needed to track time active.

**Review by acsauk — COMMENTED** [review]
Would we want to track donor [accepting access](https://github.com/ministryofjustice/opg-modernising-lpa/blob/main/internal/page/donor/enter_access_code.go) to an LPA too?

File: internal/page/sign_out.go (hunk 2 above)
Comment (hawx) [reply internal/page/sign_out.go:27]: session_id will be added here from the context, it's only on login that we need to manually add it

**Review by acsauk — APPROVED** [review]
:shipit:

**Pull request merged**